Rakefile

```diff
@@ -1,7 +1,39 @@
 require 'rake/testtask'
 
 Rake::TestTask.new do |t|
-  t.pattern = "test/**/*_test.rb"
+	t.pattern = "test/**/*_test.rb"
+end
+
+namespace :db do
+	desc "Create a new migration"
+	task :migration, [:name]  do |task, args|
+		path = "migrations/#{Time.now.to_i}_#{args[:name]}.rb"
+		File.open(path, "w") do |file|
+			file.write %{Sequel.migration do
+	change do
+	end
+end}
+		end
+	end
+
+	desc "Run migrations"
+	task :migrate, [:version] do |t, args|
+		require "sequel"
+		if !ENV["RACK_ENV"] || ENV["RACK_ENV"] == "development"
```

Comment:
If this does nothing when `.env` is not present it would be safe to use it in every environment?